### DISCUSSION TITLE
flatpak build fails when LTO

### DISCUSSION THREAD

**Discussion by ColinKinloch** [DISTRHO/Cardinal#547] [Q&A] [answered]
The flathub build fails on "Creating CLAP plugin for Cardinal" complaining about undefined references in the surgext plugin.
It builds fine when `WITH_LTO=false`.

https://buildbot.flathub.org/#/builders/26/builds/6594

**Comment by falkTX** [comment, marked as answer]
that is new, haven't seen that issue before.

in any case LTO builds are not recommended anymore, as no single Rack module developer (that Cardinal uses) uses or tests for LTO builds.
there were quite some reports of things breaking down when LTO is enabled, even though there were no symbol conflicts detected.
since the upstream modules do not support it, best to just give up on it for now.
